plugins/mattermost/server/mattermost/api.ts

```diff
@@ -0,0 +1,8 @@
+export const MattermostApi = {
```

Comment:
I would prefer to see this defined as a class with static methods, `MattermostClient` or similar – this is generally the pattern we use elsewhere.